source/_integrations/alarm_control_panel.mqtt.markdown

```diff
@@ -205,3 +205,20 @@ value_template:
   required: false
   type: template
 {% endconfiguration %}
+
+## Example
```

Comment:
Why was this example needed? There is already an initial example on the top of the documentation, right? What does this add?

Could you add some text to the paragraph explaining the example and what this example clarifies?